package/CHANGELOG

```diff
@@ -16,12 +16,13 @@ The rules for this file:
 -------------------------------------------------------------------------------
 ??/??/?? IAlibay, HeetVekariya, marinegor, lilyminium, RMeli,
          ljwoods2, aditya292002, pstaerk, PicoCentauri, BFedder,
-         kainszs
+         kainszs, tyler.je.reddy, SampurnaM
 
  * 2.8.0
 
 Fixes
  * Atomname methods can handle empty groups (Issue #2879, PR #4529)
+ * Add support for TPR files produced by Gromacs 2024.1 (PR #4523)
```

Comment:
@yuxuanzhuang I noticed a missing reference to the PR, so I added it. I hope that ist okay.

Reply:
Normally you don't have to fix up the CHANGELOG (it confuses reviewers...) but this is fine, thanks to your explanation.